pkg/cluster/nsg.go

```diff
@@ -31,7 +31,7 @@ func (m *manager) clusterNSG(infraID, location string) *arm.Resource {
 					SourceAddressPrefix:      to.StringPtr("*"),
 					DestinationAddressPrefix: to.StringPtr("*"),
 					Access:                   mgmtnetwork.SecurityRuleAccessAllow,
-					Priority:                 to.Int32Ptr(101),
+					Priority:                 to.Int32Ptr(120),
```

Comment:
I don't know if there's a reason for this or if the change is just an oversight, pointing it out in case it's the latter.

Reply:
Simply Secure NSG policies apply with a priority within the 101 range. 120 keeps it out of the range an ensures no overlap and conflict